### PR TITLE
Blogpost: introducing targetless mode

### PR DESCRIPTION
**Pull request by t4lz**
#50

### CODE REVIEW

File: content/en/blog/introducing-targetless/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: "Easy K8S Connectivity for Local Utils"
+description: "A short introduction to the new targetless mode of mirrord, explaining its main use-cases."
```
Comment (eyalb181) [inline content/en/blog/introducing-targetless/index.md:3]: Also how about adding a meme or something?

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/en/blog/introducing-targetless/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: "Easy K8S Connectivity for Local Utils"
```
[the remark this answers is not in the document]
Comment (t4lz) [reply content/en/blog/introducing-targetless/index.md:2]: Added linebreaks.

[the remark this answers is not in the document]
Comment (eyalb181) [reply content/en/blog/introducing-targetless/index.md:2]: They're not showing in the preview

[the remark this answers is not in the document]
Comment (t4lz) [reply content/en/blog/introducing-targetless/index.md:2]: Forgot to push 🤦

File: content/en/blog/introducing-targetless/index.md (hunk 1 above)
Comment (eyalb181) [reply content/en/blog/introducing-targetless/index.md:3]: For your consideration:

image

**Review by eyalb181 — CHANGES_REQUESTED** [review]
Looks good, one last thing: we're planning to publish on Monday, so please change the date to Monday morning

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**